### PR TITLE
Adding comments to config options

### PR DESCRIPTION
**Pull request by ShadedJon**
I'm unsure to a couple of the settings. Please advise.

### CODE REVIEW

**Comment by rourke750** [comment]
Can you put the comment tags above each config option.  That's how it normally is.

File: config.yml [hunk 1]
```diff
@@ -1,40 +1,67 @@
 free_world: world
+# world imprisoned players respawn to once freed
 free_respawn: true
+# if set to false: sends pearled players to free world if freed when offline. if set to true: keeps players in end if freed when offline.
 free_tppearl: true
+# if freed while online, will teleport to the player holding the pearl.
 damagelog_min: 3
```
Comment (rourke750) [inline config.yml]: I believe this is the min amount of damage needed before a pearl is awarded.

File: config.yml [hunk 2]
```diff
@@ -1,40 +1,67 @@
...
+# unsure
 damagelog_ticks: 600
```
Comment (rourke750) [inline config.yml]: The amount of ticks the damagelogger keeps track of damages done to players before resetting.

File: config.yml [hunk 3]
```diff
@@ -1,40 +1,67 @@
...
+# requires players to have enderpearl in hotbar to pearl another killed player
 summon_damage_radius: 20
+# the distance a summond player can travel by default from the pearlholder
 summon_damage_ticks: 20
+# how many ticks until damage is dealt to pearled player outside summon_damage_radius
 summon_damage_amt: 2
+# amount of damage dealt to prisoned player
 prison_world: world_the_end
+# which world players are sent to when killed by another player with an enderpearl in their hotbar (or inventory depending on prison_hotbar setting)
 prison_resetbed: false
```
Comment (rourke750) [inline config.yml]: If it should reset a player's bed on pearl.

File: config.yml [hunk 4]
```diff
@@ -1,40 +1,67 @@
...
+# unsure
 chunk_delay: 1200
```
Comment (rourke750) [inline config.yml]: The amount of time to wait if a pearl is dropped due to logging out and the chunk is trying to unload to free a pearl.

File: config.yml [hunk 5]
```diff
@@ -1,40 +1,67 @@
...
 ppsummon_clear_inventory: false
+# if true clears the inventory of a prisoned player when summoned to the free_world
 ppreturn_kills: false
+# if true, when pearled players are /ppreturned they die and respawn in the prison_World
 prison_motd:
    - You are trapped in this bleak and endless world
    - by a prison pearl.
    - Type /pplocate to locate your pearl.
+# the message that is displayed when players join the prison_world after being imprisoned
 save_ticks: 2400
```
Comment (rourke750) [inline config.yml]: The amount of ticks to wait before saving...

File: config.yml [hunk 6]
```diff
@@ -1,40 +1,67 @@
...
+# unsure
 upkeep:
+# this setting controls the 'pearl cost'. PrisonPearls require quantity amount of resource placed inside the chest the prisonpearl is located in to keep the player pearled.
   resource: 263
+# the Minecraft item ID of the resource required to keep players pearled.
   quantity: 4
+# the amount of resource taken at each restart. If there aren't enough of the resource, the pearl is released
   distance: 0
```
Comment (rourke750) [inline config.yml]: The distance from 0,0 that if a pearl exceeds is freed.

File: config.yml [hunk 7]
```diff
@@ -1,40 +1,67 @@
...
+# unsure
 ignore_feed:
   days: 0
   hours: 0
   seconds: 0
+# this setting disables the prisonpearl from taking more resources at restart until the amount of time set above has passed.
 feed_delay: 72000000
```
Comment (rourke750) [inline config.yml]: The amount of time to wait before feeding can occur again.

**Comment by ShadedJon** [comment]
@rourke750 I added your changes.

Still unsure on a couple things, I'll comment them in separate comments to this thread.

(also, still pretty new to Github, please advise me if I'm doing stuff wrong)

File: config.yml [hunk 8]
```diff
@@ -1,40 +1,67 @@
...
    - Type /pplocate to locate your pearl.
+# the amount of ticks to wait before saving
 save_ticks: 2400
+# this setting controls the 'pearl cost'. PrisonPearls require quantity amount of resource placed inside the chest the prisonpearl is located in to keep the player pearled.
 upkeep:
+# the Minecraft item ID of the resource required to keep players pearled.
   resource: 263
+# the amount of resource taken at each restart. If there aren't enough of the resource, the pearl is released
   quantity: 4
+# the max distance a pearl can be away from 0,0 before being freed.
```
Comment (ShadedJon) [inline config.yml]: why is this under upkeep, also will probably need multi-world support for sharding

File: config.yml [hunk 9]
```diff
@@ -1,40 +1,67 @@
...
+# if true clears the inventory of a prisoned player when summoned to the free_world
 ppsummon_clear_inventory: false
+# if true, when pearled players are /ppreturned they die and respawn in the prison_World
 ppreturn_kills: false
+# the message that is displayed when players join the prison_world after being imprisoned
 prison_motd:
    - You are trapped in this bleak and endless world
    - by a prison pearl.
    - Type /pplocate to locate your pearl.
+# the amount of ticks to wait before saving
```
Comment (ShadedJon) [inline config.yml]: Saving what? The pearled players list to file?

File: config.yml (hunk 8 above)
Comment (Maxopoly) [inline config.yml]: Sharding doesn't need multi-world support, it's entirely different servers and each has it's own config for every plugin

Comment (ShadedJon) [inline config.yml]: Gotcha

**Comment by Maxopoly** [comment]
Format completly changed since this PR was made, so rejecting it for now.

**Pull request closed without merging**